Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1266,6 +1298,7 @@ def _extract_directives(self, node, scope_name):
         node.decorators = realdecs[::-1] + both[::-1]
         # merge or override repeated directives
         optdict = {}
+        contents_optdict = {}
         for directive in directives:
             name, value = directive
```

Comment:
And while we're at it, there's no need for this either.
```suggestion
        for name, value in directives:
```